### DISCUSSION TITLE
Invalid configurations for domains

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#3686] [Help] [answered]
### Summary

Hi! I've added a new domain in js.cool (https://zepvil.js.cool) but it won't work. I've first tried CNAME and A but the domain just doesn't work. Help would be appreciated.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/99463015/2f125cc0-6e9f-49b3-a743-41833fd75e3d)

**Comment by amyegan** [comment, marked as answer]
Hey @Zepvil. I don't see the Vercel address when I check the A record for `zepvil.js.cool`. Instead I see `104.21.42.134` and `172.67.206.25`. You can check current values at https://www.whatsmydns.net/#A/zepvil.js.cool

Please double check that you have set the correct value with your DNS provider. Keep in mind that it can take some time for records to propagate through the global DNS network.
